ctapipe/image/pixel_likelihood.py

```diff
@@ -51,42 +51,62 @@ class PixelLikelihoodError(RuntimeError):
     pass
 
 
-def poisson_likelihood_gaussian(image, prediction, spe_width, ped):
-    """
-    Calculate likelihood of prediction given the measured signal, gaussian approx from
-    de Naurois et al 2009
+def poisson_likelihood_gaussian(image, prediction, spe_width, pedestal):
+    """Calculate negative log likelihood for every pixel.
+
+    Gaussian approximation from de Naurois, p. 22 (between (24) and (25)).
+
+    Simplification:
+
+    .. math::
+
+        θ = σ_p^2 + μ · (1 + σ_γ^2)
+
+        → P = \\frac{1}{\\sqrt{2 π θ}} · \\exp\\left(- \\frac{(s - μ)^2}{2 θ}\\right)
+
+        \\ln{P} = \\ln{\\frac{1}{\\sqrt{2 π θ}}} - \\frac{(s - μ)^2}{2 θ}
+
+                = \\ln{1} - \\ln{\\sqrt{2 π θ}} - \\frac{(s - μ)^2}{2 θ}
+
+                = - \\frac{\\ln{2 π θ}}{2} - \\frac{(s - μ)^2}{2 θ}
+
+                = - \\frac{\\ln{2 π} + \\ln{θ}}{2} - \\frac{(s - μ)^2}{2 θ}
+
+        - \\ln{P} = \\frac{\\ln{2 π} + \\ln{θ}}{2} + \\frac{(s - μ)^2}{2 θ}
+
+    and since we can remove constants and factors in the minimization:
+
+    .. math::
+
+        - \\ln{P} = \\ln{θ} + \\frac{(s - μ)^2}{θ}
+
+
+    References
+    ----------
+    - de Naurois, Rolland
+      'A high performance likelihood reconstruction of gamma-rays
+      for Imaging Atmospheric Cherenkov Telescopes' (2009)
```

Comment:
could move this to `references.rst`.  See also #1266

Reply:
done